client/my-sites/invites/p2/invite-accept-logged-out.jsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import SignupForm from 'calypso/blocks/signup-form';
+import { renderInviteAcceptP2Logo } from './invite-accept-logo-line';
+import { renderInviteAcceptFormHeader } from './invite-accept-form-header';
+import { renderInviteAcceptFooter } from './invite-accept-footer';
+
+export function renderInviteAcceptForP2( props ) {
+	return (
+		<div>
+			{ renderInviteAcceptP2Logo( { translate: props.translate } ) }
+			<div className="invite-accept-logged-out__form-container">
+				{ renderInviteAcceptFormHeader( { site: props.invite.site, translate: props.translate } ) }
+				<SignupForm
+					redirectToAfterLoginUrl={ window.location.href }
+					disabled={ props.isSubmitting }
+					formHeader=""
+					submitting={ props.isSubmitting }
+					save={ props.save }
+					submitForm={ props.submitForm }
+					submitButtonText={ props.isSubmitting ? '' : props.translate( 'Sign up and join' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( 'Sign Up & Join' )` *ES Score: 7*

Reply:
Keeping this for consistency with P2 strings.